application/src/main/java/com/softserve/lv460/application/security/filters/JwtAuthenticationFilter.java

```diff
@@ -24,8 +24,8 @@
   private JwtTokenProvider tokenProvider;
   @Autowired
   private UserDetailsServiceImpl customUserDetailsService;
-@Autowired
-private SecurityConfigProperties securityConfigProperties;
+  @Autowired
+  private SecurityConfigProperties securityConfigProperties;
 
   private static final Logger logger = LoggerFactory.getLogger(JwtAuthenticationFilter.class);
```

Comment:
we use @Slf4j

Reply:
Resolved + change @Autowired's to constructors